src/net.h

```diff
@@ -530,7 +530,7 @@ struct LocalServiceInfo {
     int nPort;
 };
 
-extern CCriticalSection cs_mapLocalHost;
+extern Mutex cs_mapLocalHost;
```

Comment:
In Bitcoin, RecursiveMutex is used.

Reply:
```
extern GlobalMutex g_maplocalhost_mutex;
```
you mean they say globalmutex which is actually
```
class GlobalMutex : public Mutex { };
```